cpp/src/io/orc/orc.hpp

```diff
@@ -84,8 +84,8 @@ struct StripeInformation {
   uint64_t offset       = 0;  // the start of the stripe within the file
   uint64_t indexLength  = 0;  // the length of the indexes in bytes
   uint64_t dataLength   = 0;  // the length of the data in bytes
-  uint32_t footerLength = 0;  // the length of the footer in bytes
-  uint32_t numberOfRows = 0;  // the number of rows in the stripe
+  uint64_t footerLength = 0;  // the length of the footer in bytes
+  uint64_t numberOfRows = 0;  // the number of rows in the stripe
```

Comment:
Sorry I missed this. It should be `int64_t` too. I'll change that in the future PR.

Reply:
These should not be int64_t. This struct is a C++ implementation of the protobuf message as defined in ORC specs. We should be aligned here, and cast to signed when we use the values.

Reply:
OK got it.